Task1/Task1/Program.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+
+namespace Task1
+{
+    class Program
+    {
+        static void Main()
+        {
+            Console.WriteLine(2 + 3);
```

Comment:
Лучше для наглядности
```suggestion
            Console.WriteLine("2 + 3 =" + (2 + 3).ToString());
```